src/js/auth/routes.jsx

```diff
@@ -0,0 +1,8 @@
+import LoginApp from './containers/LoginApp';
+
+const routes = {
+  path: '/auth/login/callback',
```

Comment:
LGTM! Note that this does not have to match the fetch call to the API in `src/js/auth/containers/LoginApp.jsx`

I will ask ID.me to update their callback urls to the following:
- local development: `saml-rp.vetsgov.localhost`, `http://localhost:3001/auth/login/callback`
- dev.vets.gov: `saml-rp.vetsgov.dev`, `https://dev.vets.gov/auth/login/callback`